src/components/info/street-view.vue

```diff
@@ -0,0 +1,27 @@
+<script setup lang="ts">
+import { useTranslation } from 'i18next-vue'
+import { storeToRefs } from 'pinia'
+import { useInfoStore } from '@/stores/info.store'
+
+const { t } = useTranslation()
+
+const { isStreetviewActive, noDataAtLocation } = storeToRefs(useInfoStore())
+</script>
+
+<template>
+  <div
+    id="streetview-div"
+    class="h-[500px]"
+    v-show="isStreetviewActive && !noDataAtLocation"
+  >
+    Streetview Container
+  </div>
+  <div
+    class="grid before:content-streetview before:col-start-1 before:row-start-1"
```

Comment:
Why is this a grid? you only display a text message.

Reply:
modern equivalent of `position: absolute`
https://dev.to/nicm42/using-css-grid-to-put-elements-on-top-of-each-other-44ei